packages/react-components/react-tabster/src/hooks/useOnKeyboardNavigationChange.ts

```diff
@@ -18,6 +18,7 @@ export function useOnKeyboardNavigationChange(callback: (isNavigatingWithKeyboar
         eventCallback(next);
       };
       keyborg.subscribe(cb);
+      cb(keyborg.isNavigatingWithKeyboard());
```

Comment:
Would probably be good for @ling1726 to review this change

Reply:
Yeah would love to know if there are some unexpected side effects of doing this. The issue being solved here is the Listbox subscribing to keyborg events after keyborg has initialized and already set "isNavigatingWithKeyboard" to true, so the listbox has no idea that it is true already. From my looks around the usages, this _should_ be safe, but I am no expert here :)

Reply:
Yeah I agree that initial subscription should provide the correct value. Let's keep this here for now. Ideally keyborg should be doing this